specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2019-06-15/examples/AzureIaasVm/ProtectionPolicies_CreateOrUpdate_Complex.json

```diff
@@ -4,7 +4,7 @@
     "resourceGroupName": "SwaggerTestRg",
     "vaultName": "NetSDKTestRsVault",
     "policyName": "testPolicy1",
-    "api-version": "2019-06-15",
+    "api-version": "2020-12-01",
```

Comment:
I meant update the api-versions for the files in the new folder, not existing folders

Reply:
My bad, reverted the commit and added a new one only for new API version